examples/resources/awscc_backup_backup_selection/fourth_example.tf

```
@@ -0,0 +1,18 @@
+resource "awscc_backup_backup_selection" "example4" {
```

```suggestion
resource "awscc_backup_backup_selection" "example" {
```
no need to create increment